CMakeLists.txt

```
@@ -72,5 +83,7 @@ packageProject(
   INCLUDE_DIR ${PROJECT_SOURCE_DIR}/include
   INCLUDE_DESTINATION include/${PROJECT_NAME}-${PROJECT_VERSION}
   VERSION_HEADER "${VERSION_HEADER_LOCATION}"
-  DEPENDENCIES ""
+  COMPATIBILITY ExactVersion
+  # TBD COMPATIBILITY SameMajorVersion
```

I think we can use `SameMajorVersion` and remove the comment.

```suggestion
  COMPATIBILITY SameMajorVersion
```